### PR TITLE
Created Results Page

### LINKED ISSUE #2: Create static Restaurant-Result-Page
## Value statement - a requirement, not a solution
As a user you will find all the results of your search inside this page collected inside a list

## Description (Text, Scribble, Wireframe, Design)
Screenshot 2021-04-26 at 11 00 56

## Acceptance criteria
- Once Through the filter process the results page will show all results
- The results will be listed underneath each other
- The button will redirect to beginning of filtering

## Tasks
- Create headline
- Create a box which displays the content 
- Add fox layout for multiple boxes
- Add a photo and adjust size to fit inside the box
- Add Svg´s as the icons
- Add description

## Size
medium

### PR DESCRIPTION
**Pull request by soerendolling**
- I have created the Results Page
- use /resultsPage afterUrl to check in Vercel

### CODE REVIEW

File: src/styles/fonts/fonts.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
   --subheading-size: 20px;
   --subheading-color: var(--gold-color);
   --subheading-spacing: 4px;
+  --subheading-weight: 300;
   --mainButton-size: 25px;
   --mainButton-color: var(--background-color);
   --mainButton-spacing: 2px;
```
Comment (yvlange) [inline src/styles/fonts/fonts.css:12]: maybe put all the colors you use in one folder and the font-styling (size, spacing) in this folder

File: src/components/ResultsPage.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+a {
+  text-decoration: none;
+}
+
+.results-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
```
Comment (yvlange) [inline src/components/ResultsPage.css:10]: I could see that your header has a height of 160px. I think you can reduce the height to 100px so the content has a bit more room :)

**Review by yvlange — APPROVED** [review]
Looks good and clear to me :) only two tiny comments

File: src/styles/fonts/fonts.css (hunk 1 above)
Comment (soerendolling) [reply src/styles/fonts/fonts.css:12]: True, Got one color.css maybe i should just define it there.

**Pull request merged**